site/ui/components/shared/Navbar.vue

```diff
@@ -15,6 +15,10 @@
       <b-navbar-brand class="above-breadcrumbs" to="/">
         <span>Geney</span>
       </b-navbar-brand>
+      <b-navbar-nav class="above-breadcrumbs ml-auto">
+        <b-nav-item href="/about" class="pr-2">About</b-nav-item>
```

Comment:
We should make this a `router-link` so that the whole page doesn't reload when we go to `/about`. Let me know if you have questions about this.